puffin/src/trace.rs

```diff
@@ -680,7 +942,12 @@ impl<PT: ProtocolTypes> OutputAction<PT> {
         }
     }
 
-    fn execute<PB>(&self, agent_name: AgentName, ctx: &mut TraceContext<PB>) -> Result<(), Error>
+    fn execute<PB>(
```

Comment:
Please document the step argument. It was unclear to me at first sight.

Reply:
done